### PR TITLE
POM-407 Zmiana wyglądu przycisku usuń na wynikach wyszukiwania oraz o…

### PR DESCRIPTION
**Pull request by broslukasz**
Zmiana wyglądu przycisku usuń na wynikach wyszukiwania oraz obszaru navigacji do ogłoszenia

### CODE REVIEW

**Comment by broslukasz** [comment]
image

Here you can see how delete looks like. I think it is until edit is implemented

**Review by neu5 — DISMISSED** [review]

File: src/app/find-help/search-result/search-result.component.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <div class="right opacity-50" *ngIf="posted">{{ postedDate | date }}</div>
   </div>
   <div class="d-sm-flex">
-    <a role="button" (click)="onViewOffer()" class="flex-fill link">
+    <a role="button" class="flex-fill link">
```
Comment (mgol) [inline src/app/find-help/search-result/search-result.component.html:18]: This should not be `<a>`, especially with `role="button"`, when there's no link or handler here.

File: src/app/find-help/search-result/search-result.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<mat-card class="result">
+<mat-card class="result" (click)="onViewOffer()">
```
Comment (mgol) [inline src/app/find-help/search-result/search-result.component.html:1]: This should have a proper link in `<a>` and with a `href` attribute, perhaps generated by the router.

Would it be possible to wrap the whole card contents with `<a>` or - if providing a link here is impractical at the moment - a `<button>`?

File: src/app/my-account/my-account/my-account.component.html [hunk 3]
```diff
@@ -44,8 +44,8 @@ <h1 class="header-2 font-darkblue fw-bold">{{ "MY_ACCOUNT" | translate }}</h1>
             icon="translate"
           ></app-search-result-attribute>
 
-          <div class="pt-2 pt-sm-0 ps-sm-2" buttons>
-            <button type="button" class="btn btn-link btn-sm" (click)="removeAnnouncement(announcement)">
+          <div class="pt-2 pt-sm-0 ps-sm-2" buttons (click)="removeAnnouncement(announcement, $event)">
+            <button type="button" class="btn btn-light btn-sm">
```
Comment (mgol) [inline src/app/my-account/my-account/my-account.component.html:48]: Button without a `click` handler and div with one. I know we're cutting corners but this seems like too much... CSS can be adjusted if that's the issue.

File: src/app/find-help/search-result/search-result.component.html (hunk 2 above)
Comment (broslukasz) [reply src/app/find-help/search-result/search-result.component.html]: I put all in a but without href

File: src/app/my-account/my-account/my-account.component.ts [hunk 4]
```diff
@@ -62,13 +62,13 @@ export class MyAccountComponent implements OnInit {
...
 
-  removeAnnouncement(announcement: AccommodationOffer | MaterialAidOffer | TransportOffer): void {
+  removeAnnouncement(announcement: AccommodationOffer | MaterialAidOffer | TransportOffer, $event: MouseEvent): void {
+    $event.stopPropagation();
     const dialogRef: MatDialogRef<ConfirmRemoveAdComponent> = this.dialog.open(ConfirmRemoveAdComponent, {
       hasBackdrop: true,
       width: '100%',
       maxHeight: '450px',
       maxWidth: '720px',
-      disableClose: true,
```
Comment (broslukasz) [inline src/app/my-account/my-account/my-account.component.ts:71]: Revert of disableClose default value, but keept hasBackdrop.

**Pull request closed without merging**